linux-build/build-linux.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:22.10
+
+RUN apt-get update && apt-get install -y \
+    git make g++ libpng-dev cmake libspdlog-dev sudo
```

Comment:
Is sudo required ?

Reply:
not anymore :D 
I'm 99% sure it is a leftover from a different approach when trying to build this